src/StackExchange.Exceptional.AspNetCore/ConfigSettings.LogFilters.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Runtime.Serialization;
+
+namespace StackExchange.Exceptional
+{
+    internal partial class ConfigSettings
+    {
+        public LogFilterSettings LogFilters { get; set; } 
+
+        public class LogFilterSettings
+        {
+            public List<LogFilter> Form { get; set; } = new List<LogFilter>();
+
+            public List<LogFilter> Cookies { get; set; } = new List<LogFilter>();
```

Comment:
Awesome :) had this in when I was starting with the Settings impl, but removed it again because I wanted to be as close to the original as possible :) back in in b1a1513